src/shared_modules/content_manager/testtool/main.cpp

```diff
@@ -19,14 +25,15 @@
  * @outputFolder: if defined, the content will be downloaded to this folder.
  * @dataFormat: Format of the content downloaded or after decompression.
  * @fileName: Name of the file to download or file name in case of raw content.
- * @offset (integer): Api offset used to override (if greater) the one set on the database.
  */
 static const nlohmann::json CONFIG_PARAMETERS =
     R"(
         {
             "topicName": "test",
             "interval": 10,
             "ondemand": true,
+            "createDatabase": true,
+            "initialOffset": "0",
```

Comment:
Remove these configuration and have the database created each time testtool is run.